spec/events_spec.rb

```diff
@@ -0,0 +1,110 @@
+require_relative "spec_helper"
+
+RSpec.describe Courier::Events do
+  let(:client) { Courier::Client.new(AUTH_TOKEN_MOCK) }
```

Comment:
I would refrain from putting constants that are autoloaded into the global namespace by requiring spec_helper, but it works well enough for your needs here. To work around it elegantly you'd need some sort of factory solution and that is out of scope for this work.